Documentation/Guides/SSL Pinning.md

```diff
@@ -0,0 +1,36 @@
+### SSL Pinning
```

Comment:
I'm not sure if there's a hard standard on this, but I think the markdown `#` characters are supposed to be functionality equivalent to HTML header tags (e.g. `<h1>`, `<h2>`, etc.). I think I've read somewhere that it's good practice to always have one top-level "h1" level header and then go from there. So, since this is a fresh document/page, we could follow that pattern to start with a single `#` here at the top level and then fill out the rest of the titles accordingly.

```markdown
# SSL Pinning
...

## iOS 14+
...

## Older iOS Versions
...
```

Reply:
That's a fair point - I had basically just matched what existed already, even if it was "wrong". I went back through the documentation and applied similar formatting.